apps/site/assets/ts/stop/components/StopPageRedesign.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { ReactElement } from "react";
+
+const StopPageRedesign = ({
+  stopId
+}: {
+  stopId: string;
+}): ReactElement<HTMLElement> => {
+  return (
+    <div>
+      {/* Title Bar Div */}
+      <div className="d-flex justify-content-space-between">
+        <div>
+          <h1>{stopId}</h1>
+          {/* ICONS GO HERE */}
+        </div>
+        <div style={{ marginTop: "3.075rem" }}>
+          Zone Information PLACEHOLDER
+        </div>
+      </div>
+      {/* Route and Map Div */}
+      <div className="d-flex">
+        <div style={{ minWidth: "50%" }}>
```

Comment:
This is okay for managing the trips/map layout, but there's room for improvement. 

**suggestion:** When we get small enough to hit the `sm` breakpoint, we're left with a void where the map was (pictured below) – it would be better to let the departures list take full width here.
Maybe try using CSS grid? `display: grid; grid-template-columns: 1fr auto;` on the wrapping div will show the departures and map side by side but also let departures take full width once the map is hidden.

<img width="849" alt="image" src="https://user-images.githubusercontent.com/2136286/215186268-244f4d51-5014-4f63-841a-28717463a98d.png">

Reply:
I am hesitant to spend too much time styling this.  As the styling feels really up in the air still, design and coding wise.
I think the CSS should be tweaked after more of the child components get coded.

For this specific case I think it looks like display width just needs to be set to 100%.  Also that wouldn't necessarily fix the void, as there is nothing to fill the white space (that list stacks).  With more filled in components it would be better to see how to fix that void.  (Maybe center the list idk).